feature-configs/demo/performance/machine_config_pool.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-rt
+  namespace: openshift-machine-config-operator
+  labels:
+    machineconfiguration.openshift.io/role: worker-rt
+spec:
+  paused: true
+  machineConfigSelector:
+    matchExpressions:
+      - key: machineconfiguration.openshift.io/role
+        operator: In
+        values: [worker,worker-rt]
```

Comment:
@fedepaol @slintes this approach is different from what we have in https://github.com/openshift-kni/cnf-features-deploy/pull/25

here, the mcp is being delivered as part of the manifests. In #25 the mcp is being delivered in the pre setup logic before features are deployed.

how would you all feel about creating a new subdirectory for the MCP config since it's going to be shared between multiple features? The pre setup logic would label the nodes, and the MCP config would be deployed as part of the FEATURES loop. everything else should fall inline from there eventually. 

@fedepaol and I were discussing giving this MCP a more generic name, like worker-cnf, so it would make more sense when we apply multiple cnf related features to it.

Reply:
Sorry, didn't spend much attention to other PRs, just wanted to update the operator over here after the corresponding PR was finally merged.
CI is green, and I see worker-rt MC and MCP, it seems to basically work.
I don't mind if we merge this and rebase others, or just pick relevant changes from here to other PRs and close this.

Reply:
I only have one question: my understanding was that the operator would take the mcp name as an argument, but I am not seeing it happening as part of this PR.
So the question is, how does it work?

Reply:
Looked into the code, it uses the nodes selector as the mcp selector if not specified, right? 
https://github.com/openshift-kni/performance-addon-operators/blob/e5c054f608d2fd21611f267e71eed51289c53e9a/pkg/controller/performanceprofile/components/profile/profile.go#L73

Reply:
Exactly, it takes the role from the node selector 👍

Reply:
> Looked into the code, it uses the nodes selector as the mcp selector if not specified, right?

@fedepaol yep yep, and for further clarification. this is why i requested that worker-cnf be consistent across nodeSelector MC and MCPs.

Reply:
@davidvossel yes I thought it was because of that